packages/run-protocol/src/runStake/attestation.js

```diff
@@ -95,16 +112,13 @@ const makeAttestationIssuerKit = async (zcf, stakeBrand, lienBridge) => {
     );
   };
   const assertAccountStateOK = async (address, toLien) => {
-    const s = await E(lienBridge).getAccountState(address, stakeBrand);
+    const s = await getAccountState(address, stakeBrand);
     const unlocked = subtractOrZero(s.total, s.locked);
     checkDelta(toLien, 'unliened', s.total, s.liened);
     checkDelta(toLien, 'bonded and unliened', s.bonded, s.liened);
     checkDelta(toLien, 'unlocked and unliened', unlocked, s.liened);
   };
```

Comment:
How expensive are these checks? If cheap then better to include in `getAccountState` so they're validated whenever it's accessed.

Reply:
Um... that's what I did, no?